src/app/language.js

```diff
@@ -158,7 +176,8 @@ App.Localization.langcodes = {
     },
     'ca': {
         name: 'Catalan',
-        nativeName: 'Català'
+        nativeName: 'Català',
+        flag: 'catalan'
```

Comment:
so 2 letter codes come from the flag library, the full names are the ones we provide. for arabic it was confusing as ar = argentina ? es-ar ? ar... some would say that catalan has nothing to do with spanish... that said we have catalan and occitan, and i'm pretty sure fr-oc is wrong.